src/functions-and-arrays.js

```diff
@@ -1,41 +1,168 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(numA, numB) {
+  let largestNumber
```

Comment:
You could do

`return Math.max(numA, numB)`

Reply:
:D that will come in handy! Thanks.